### PR TITLE
TLS-correction 3

### PR DESCRIPTION
**Pull request by jen3andruska**
Not sure why these did not go through in the 1st pull request.

### CODE REVIEW

File: docs/reference/contributing/target/entropy.md [hunk 1]
```diff
@@ -88,9 +88,9 @@ To indicate that the target has an entropy source, you have to add `TRNG` to the
...
+If a hardware platform does not have a hardware entropy source to leverage into the entropy pool, alternatives have to be considered. As stated above, a strong entropy source is crucial for security of cryptographic and TLS operations. For platforms that support non-volatile memory, an option is to use the NV seed entropy source that Mbed provides.
 
-This makes Mbed TLS use a fixed amount of entropy as a seed and update this seed each time entropy is gathered with an Mbed TLS entropy collector for the first time. In a simple case it means that the seed is updated after reset at the start of the first TLS connection.
+This makes Mbed use a fixed amount of entropy as a seed and update this seed each time entropy is gathered with an Mbed entropy collector for the first time. In a simple case it means that the seed is updated after reset at the start of the first TLS connection.
```
Comment (AnotherButler) [inline docs/reference/contributing/target/entropy.md]: Maybe "This uses a fixed amount of entropy as a seed and updates this seed each time an entropy collector gathers entropy for the first time."?

**Comment by jen3andruska** [comment]
@AnotherButler I think these rewordings make sense, but probably good to have an engineer review them from a content perspective.  I've added Darryl Green.

**Comment by AnotherButler** [comment]
@dgreen-arm What's the status on helping us update this file/these three PRs?

**Comment by dgreen-arm** [comment]
@AnotherButler I think these re-wordings are OK. The guide should probably keep the Mbed TLS references in the rest of the document, as it is specific to Mbed TLS. The section under `Why Mbed TLS needs entropy` does give some indication that entropy is needed for more than just TLS, but it might not be immediately apparent with its current heading, the re-wording should help with that.

**Comment by AnotherButler** [comment]
@dgreen-arm I've made those rewordings. Is this OK to merge to you?

**Comment by dgreen-arm** [comment]
Yes, I think these rewordings are fine.

**Pull request merged**